src/transformers/models/xlnet/modeling_xlnet.py

```diff
@@ -1081,7 +1080,6 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
-        **kwargs,  # delete after depreciation warning is removed
```

Comment:
I think it will break any other usage of this architecture, isn't it?

Reply:
I had to do this because `**kwargs` breaks the onnx export, as I mentioned in the PR description. It did pass all the unit test and I think the deprecation warning has been up for a while.